automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Collapse.scala

```diff
@@ -5,37 +5,41 @@ import org.openqa.selenium.WebDriver
 
 case class Collapse[A <: FireCloudView](queryString: QueryString, private val inner: A)(implicit webDriver: WebDriver)
   extends Component(queryString) with Stateful {
-  private val toggleComponent = findInner("toggle")
+
+  private val toggleCssQuery: CssSelectorQuery = findInner("toggle")
+  private val toggleLink = Link(CSSQuery(toggleCssQuery.queryString))
 
   override def awaitReady(): Unit = inner.awaitReady()
 
   def isExpanded: Boolean = {
-    awaitVisible()
-    stateOf(toggleComponent) == "expanded"
+    await condition { toggleLink.isVisible && toggleLink.isEnabled }
+    stateOf(toggleCssQuery) == "expanded"
   }
 
   def toggle(): Unit = {
-    await visible toggleComponent
-    click on toggleComponent
-    awaitReady()
+    val state = isExpanded
+    toggleLink.doClick()
+    // compare data-test-state after click
+    await condition(state != isExpanded, 15)
```

Comment:
capture value of `data-test-state` before and after click, two values should not be equal.